phing/build.yml

```diff
@@ -58,9 +60,9 @@ drupal:
 
 drush:
   bin: ${composer.bin}/drush
-  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${multisite.name}
+  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${site.uri.${environment}}
```

Comment:
@kylebrowning I haven't tried it yet, but just looking at the club alias command it doesn't appear to be multisite aware. e.g. it would generate one alias for each combination of subscription and environment, rather than one alias for each combination of _site_ (i.e. database) and environment.